todo_project_name/rsa.py

```diff
@@ -61,7 +86,8 @@ def save_key(key: RSAKey, path: Path) -> Path:
     kind = key.__class__.__name__
     header = f"-----BEGIN {kind} KEY-----"
     footer = f"-----END {kind} KEY-----"
-    contents = "\n".join((header, str(key.key), str(key.modulus), footer))
+    # TODO: Consider serialization method allowing white space in `RSAKey.id`. Is it needed?
```

Comment:
That probably will be resolved when  standard encoding will be implemented.

Reply:
Yes, I suppose. May this note serve as a reminder.